.github/scripts/test_force_reorg.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eu
+
+pushd local-tests/
+
+if [ ! -f "$ALEPH_NODE_BINARY" ]; then
```

Comment:
use some default value for `ALEPH_NODE_BINARY`. Echo the name of the missing var below?

Reply:
done